doc/filters/format_currency.rst

```diff
@@ -74,4 +134,16 @@ Arguments
 
 * ``currency``: The currency
 * ``attrs``: A map of attributes
-* ``locale``: The locale
+* ``locale``: The locale (only 'en' supported at the moment)
+
+.. note::
+
+    You can find the comprehensive list of available currencies for this filter at the following link:
+
+    https://github.com/symfony/polyfill-intl-icu/blob/master/Resources/currencies.php
```

Comment:
I would not use this file as link. The NumberFormatter API accepts currencies as the 3 letter code of ISO 4217, which is a standard. This is what we should document.